src/change/sdp.rs

```diff
@@ -1045,36 +1032,20 @@ fn update_media(
             .map(|r| r.ssrc);
         let rid = i.rid;
 
-        if let Some(existing_stream) = streams.stream_rx_by_mid_rid(mid, rid) {
-            if existing_stream.ssrc() == i.ssrc && existing_stream.rtx() == repair_ssrc {
-                continue;
-            }
-
-            // The remote changed its mapping, this happens in Firefox when transitioning an m-line
-            // from `sendonly` to `inactive` and back to `sendonly`. Firefox changes the SSRCs for
-            // simulcast layer(although not for the lowest quality layer).
-            log_change(
-                format!("Reconfiguring pre-communicated SSRC: {}", i.ssrc),
-                repair_ssrc,
-                mid,
-                i.rid,
-            );
-            let previous_ssrc = existing_stream.ssrc();
-
-            existing_stream.change_ssrc(i.ssrc);
+        if let Some(stream) = streams.stream_rx_by_mid_rid(mid, rid) {
+            // For existing streams, there might be an SSRC and RTX change.
+            // This happens with FF when toggling an m-line such as
+            // SendRecv -> Inactive -> SendRecv.
             if let Some(repair_ssrc) = repair_ssrc {
-                existing_stream.maybe_reset_rtx(repair_ssrc)
+                stream.maybe_reset_rtx(repair_ssrc)
             }
-
-            streams.change_stream_rx_ssrc(previous_ssrc, i.ssrc);
+            let from = stream.ssrc();
+            streams.change_stream_rx_ssrc(from, i.ssrc);
         } else {
-            log_change(
-                format!("Adding pre-communicated SSRC: {}", i.ssrc),
-                repair_ssrc,
-                mid,
-                rid,
+            info!(
+                "Adding pre-communicated SSRC: {:?} RTX: {:?} MID: {}",
```

Comment:
This no longer logs rid, I was trying to be consistent with the other places were we log these SSRC association events

Reply:
Good point. Let's add that.